draft-ietf-httpbis-digest-headers.md

```diff
@@ -999,6 +1003,9 @@ the transport layer that protects HTTP fields.
 A `Digest` field using NOT RECOMMENDED digest-algorithms SHOULD NOT be used in
 signatures.
 
+Protecting with a signature the `Digest` field of a request which has no payload body
+ensures that an eventual payload has not been stripped off nor been added.
```

Comment:
I don't quite follow what this is trying to say. To rephrase it based on my understanding, is it this?

"Using signatures to protect the `Digest` of an empty representation allows receiving endpoints to detect if an eventual payload has been stripped or added."

Reply:
done.